omnigibson/scene_graphs/graph_builder.py

```diff
@@ -1,10 +1,10 @@
 import itertools
 
 import networkx as nx
+import numpy as np
```

Comment:
Somehow the numpy pre-commit checker isn't working. Will need to look into it.

Reply:
manually fixed this for now